library/Haste/Form/Form.php

```diff
@@ -478,7 +478,22 @@ function($varValue, \Widget $objWidget, Form $objForm) use ($arrDca, $strName) {
             );
         }
 
-        $arrDca = $strClass::getAttributesFromDca($arrDca, $arrDca['name'], $arrDca['value']);
+        $objDca = null;
+        $strTable = '';
+
+        if (null !== ($objModel = $this->getBoundModel())) {
```

Comment:
Should we rather check for `instanceof \Contao\Model` to make sure `getTable()` method etc. is available?

Reply:
That should not be necessary, as you can only pass a `\Contao\Model` instance in `Form::bindModel`: https://github.com/codefog/contao-haste/blob/master/library/Haste/Form/Form.php#L540